src/components/QuotesText.tsx

```diff
@@ -1,17 +1,15 @@
 import React, { useState } from "react";
 import Image from "next/image";
 
-type quotes = {
+type QuotesObjProps = {
```

Comment:
Could just be Quotes, this name can be a little confusing

Reply:
addressed.